### ISSUE TITLE
pulp-glue version '0.27.0' is not in '>=0.20.0,<0.27'

### ISSUE THREAD

**Issue by UnixxSH** [pulp/squeezer#170]
Creating a container repository returns the following error: 

`An exception occurred during task execution. To see the full traceback, use -vvv. The error was: ImportError: Installed 'pulp-glue' version '0.27.0' is not in '>=0.20.0,<0.27'.
fatal: [repo]: FAILED! => {"changed": false, "msg": "Failed to import the required Python library (pulp-glue) on repo's Python /usr/bin/python3.12. Please read the module documentation and install it in the appropriate location. If the required library is installed, but Ansible is using the wrong Python interpreter, please consult the documentation on ansible_python_interpreter"}`

Versions:
pulp.squeezerr 0.0.16
ansible core 2.17.2
python 3.12.4

**Comment by mdellweg** [comment]
Yes, dependency handling on python packages is poor in ansible. But the error message actually tells you what to do: Install pulp-glue in a version >=0.20.0,<0.27 into the python environment ansible executes your plugins in. Probably the latest available 0.26.z is the best to use.

**Comment by UnixxSH** [comment]
> Yes, dependency handling on python packages is poor in ansible. But the error message actually tells you what to do: Install pulp-glue in a version >=0.20.0,<0.27 into the python environment ansible executes your plugins in. Probably the latest available 0.26.z is the best to use.

pulp-glue 0.27 is a dependency of pulp-cli 0.27. Should I also downgrade pulp-cli ? Or just ignore the dependency warning ?

**Comment by mdellweg** [comment]
If you can, I would suggest using different virtual environments. If you cannot, then yes, the version of pulp-cli and pulp-glue are locked together (only glue installs without cli if needed).
